docs/intro/tutorial05.txt

```diff
@@ -180,7 +180,7 @@ Put the following in the ``tests.py`` file in the ``polls`` application:
             """
             time = timezone.now() + datetime.timedelta(days=30)
             future_question = Question(pub_date=time)
-            self.assertEqual(future_question.was_published_recently(), False)
+            self.assertFalse(future_question.was_published_recently())
```

Comment:
I'm not convinced about the merits of `assertTrue`/`False` since such assertions pass if `bool(val)` is `True`/`False`.